cmd/sqlcmd/sqlcmd.go

```diff
@@ -10,11 +10,14 @@ import (
 
 	"github.com/alecthomas/kong"
 	"github.com/microsoft/go-mssqldb/azuread"
+	"github.com/microsoft/go-sqlcmd/internal/localizer"
```

Comment:
>-sqlcmd/internal/localizer" [](http://example.com/codeflow?start=25&length=27)

there's no localizer in the PR

Reply:
I see Localizer in internal/localizer folder